charts/rancher-logging/0.0.1/charts/fluentd/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRoleBinding
```

Comment:
`(Cluster)Role`, `(Cluster)RoleBinding`, `ServiceAccount` always work together to provide an RBAC function, so merging them into one file named `rbac.yaml` will be more readable.

Reply:
The rbac.yaml may too long, in rancher/chart and helm/charts some charts also separated, I prefer not merge